reaktive-test/src/commonTest/kotlin/com/badoo/reaktive/test/DummyTest.kt

```diff
@@ -0,0 +1,10 @@
+package com.badoo.reaktive.test
+
+import kotlin.test.Test
+
+class DummyTest {
```

Comment:
DummyTest?

Reply:
There was some problem with JS tests as there were no tests in module. Removed DummyTest as JS tests are switched off for now. You will probable need to add it back.